src/Components/ErrorBoundary/ErrorBoundary.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+export class ErrorBoundary extends React.Component {
+  state = {
+    hasError: false
+  }
+
+  componentDidCatch(error, errorInfo) {
```

Comment:
To configure "ErrorBoundary" correctly we need to add static method along with `componentDidCatch`  as well `static getDerivedStateFromError(error) {
    return { hasError: true };
  }